packages/meteor-developer-config-ui/package.js

```diff
@@ -0,0 +1,13 @@
+Package.describe({
+  summary: 'Blaze configuration templates for the Meteor developer accounts OAuth.',
+  version: '1.0.0-beta.1'
```

Comment:
Just so it's clear, the actual release process for experimental releases (beta/RC) entails updating `-beta.`/`-rc.` suffixes for packages which are changed in that release.  This means PRs need not be submitted with the suffixes included (or they _can_ be, but unless timing is perfect, it'll likely have to be changed anyway).  Bumping the version number is the important/helpful part. :smile: